src/components/TempTodo/TempTodo.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  isLoading: boolean;
+};
+
+export const TempTodo: React.FC<Props> = React.memo(({
+  todo,
+  isLoading,
+}) => {
+  const { title, completed } = todo;
+
+  const [isEdited] = useState(false);
+
+  const editFormRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    if (editFormRef.current && isEdited) {
+      editFormRef.current.focus();
+    }
+  }, [isEdited]);
+
+  return (
+    <div
+      className={cn('todo', {
+        completed,
+      })}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      {isEdited
+        ? (
+          <form>
+            <input
+              type="text"
+              className="todo__title-field"
+              placeholder="Empty todo will be deleted"
+              value={title}
+            />
+          </form>
+        )
+        : (
+          <>
+            <span
+              className="todo__title"
+            >
+              {title}
+            </span>
+
+            <button
+              type="button"
+              className="todo__remove"
+            >
+              {'\u00d7'}
+            </button>
+          </>
+        )}
```

Comment:
you don't need it into todo
```suggestion
            <span
              className="todo__title"
            >
              {title}
            </span>

            <button
              type="button"
              className="todo__remove"
            >
              {'\u00d7'}
            </button>
        )}
```